app/assets/stylesheets/style.scss

```diff
@@ -10348,20 +10087,47 @@ background: linear-gradient(180deg, rgba(255,252,252,0) 0%, rgba(126,109,109,0.5
 }
 .profile-game-list {
   height: 50vh;
+  width: 100%;
   bottom: 5vh;
   overflow: scroll;
+  color: #949494;
   // margin-bottom: 50px;
 }
+
 .saved-games{
   // width: 100vw;
   border-bottom: 3px solid gray;
   // color: #e9c318 !important;
   font-style: italic;
 }
-.game-table th {
-  // background: gray;
-  font-style:italic;
+
+.my-games {
+  width: 100%;
```

Comment:
I had to create another class because 100% looked weird on the game library page but looks better on the profile so both tables require different width requirements